### ISSUE TITLE
combine_tensor_patches produces shape error when recombining patches

### ISSUE THREAD

**Issue by siddu9501** [kornia/kornia#2973]
### Describe the bug

combine tensor patches throwing shape error when used with patch sizes greater than 32
kornia version 0.7.0

### Reproduction steps

```bash
kernel = 64
out = extract_tensor_patches(torch.arange(1024*1024*3).view(1, 3, 1024, 1024), window_size=kernel, stride=kernel)
print(out.shape)
# prints the correct shape here: (1, 16, 3, 256, 256)
combine_tensor_patches(out, original_size=(1024, 1024), window_size=kernel, stride=kernel)
```

### Expected behavior

Expected:
Tensor of shape (1, 3, 1024, 1024)
and matching the original tensor

Got the below error:
```
---------------------------------------------------------------------------
RuntimeError                              Traceback (most recent call last)
Cell In[74], line 4
      2 out = extract_tensor_patches(torch.arange(1024*1024*3).view(1, 3, 1024, 1024), window_size=kernel, stride=kernel)
      3 print(out.shape)
----> 4 combine_tensor_patches(out, original_size=(1024, 1024), window_size=kernel, stride=kernel).shape
File /mnt/xarfuse/uid-27928/f5925169-seed-nspid4026534555_cgpid10245306-ns-4026534552/kornia/contrib/extract_patches.py:277, in combine_tensor_patches(patches, original_size, window_size, stride, unpadding)
    270         raise NotImplementedError("Insufficient padding")
    272     window_size = (
    273         (original_size[0] + (unpadding[2] + unpadding[3])) // window_size[0],
    274         (original_size[1] + (unpadding[0] + unpadding[1])) // window_size[1],
    275     )
--> 277 patches_tensor = patches.view(-1, window_size[0], window_size[1], *patches.shape[-3:])
    278 restored_tensor = concatenate(torch.chunk(patches_tensor, window_size[0], 1), -2).squeeze(1)
    279 restored_tensor = concatenate(torch.chunk(restored_tensor, window_size[1], 1), -1).squeeze(1)
RuntimeError: shape '[-1, 64, 64, 3, 64, 64]' is invalid for input of size 3145728
```

### Environment

```shell
wget https://raw.githubusercontent.com/pytorch/pytorch/main/torch/utils/collect_env.py
# For security purposes, please check the contents of collect_env.py before running it.
python collect_env.py
```
- PyTorch Version (e.g., 1.0):
- OS (e.g., Linux):
- How you installed PyTorch (`conda`, `pip`, source):
- Build command you used (if compiling from source):
- Python version:
- CUDA/cuDNN version:
- GPU models and configuration:
- Any other relevant information:
```


### Additional context

_No response_

**Comment by edgarriba** [comment]
@siddu9501 are you open to send a PR to fix this error ?

**Comment by siddu9501** [comment]
@edgarriba This seems to have been fixed in the latest version of kornia (0.7.3) ; verified the change and it seems to work as expected. Apologies for raising this as a bug.
